src/sentry/static/sentry/app/stores/hookStore.jsx

```diff
@@ -18,6 +18,8 @@ const validHookNames = new Set([
   'metrics:event',
 
   // Specific component customizations
+  'component:onboarding-survey-view',
```

Comment:
can we put multiple components in the same key since they're logically related? (the hook allows you to return a list)